static/js/components/CandidateList.jsx

```diff
@@ -143,12 +231,33 @@ const CandidateList = () => {
   useEffect(() => {
     if (candidates === null) {
       setQueryInProgress(true);
-      dispatch(candidatesActions.fetchCandidates());
+      dispatch(
+        candidatesActions.fetchCandidates({ numPerPage: defaultNumPerPage })
+      );
     } else {
       setQueryInProgress(false);
     }
   }, [candidates, dispatch]);
 
+  const candidateHasAnnotationItem = (candidateObj) => {
```

Comment:
```suggestion
  const candidateHasAnnotationWithSelectedKey = (candidateObj) => {
```
or is that not what's happening here? The naming could be clearer, perhaps with some in-line comments as well